test/e2e/keycloak_client_test.go

```diff
@@ -78,7 +80,7 @@ func getKeycloakClientCR(namespace string, external bool) *keycloakv1alpha1.Keyc
 				PublicClient:              true,
 				FrontchannelLogout:        false,
 				Protocol:                  "openid-connect",
-				FullScopeAllowed:          true,
+				FullScopeAllowed:          &fullScopeAllowed,
```

Comment:
Could you please inline this? The simplest way is to create an array and point its first element. Ugly but works.

Reply:
Sure, thanks for the tip. Applied to https://github.com/keycloak/keycloak-operator/pull/286, waiting for merge+rebase here.